tests/migration_test.go

```diff
@@ -2271,7 +2257,15 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				}
 
 				_, err := virtClient.MigrationPolicy().Create(context.Background(), migrationPolicy, metav1.CreateOptions{})
-				Expect(err).ToNot(HaveOccurred())
+				return err
+			}
+
+			table.DescribeTable("should be able to cancel a migration", func(createVMI vmiBuilder, with_virtctl bool) {
```

Comment:
Another nice fact is that we don't need to use fedora here anymore afaik.

Reply:
Yep can do this in a follow up 😊